tests/xtro-sharpie/common-ObjCRuntime.ignore

```diff
@@ -38,3 +38,8 @@
 ## only on iOS and macOS
 !unknown-pinvoke! object_getInstanceVariable bound
 !unknown-pinvoke! object_setInstanceVariable bound
+
+## present in <os/log.h>
+!unknown-pinvoke! os_log_create bound
```

Comment:
It's actually in Kernel.framework.

Reply:
@Therzok I think you meant System.framework. Kernel.framework is usable in KEXTs only.

Reply:
I grepped for os_log_create in `/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform`. But yeah, seems like System.framework! Can we make xtro know about this and not have to do ignore it @rolfbjarne ?

Reply:
Gotcha!